.github/workflows/build-timestamped-master.yml

```
@@ -3,46 +3,17 @@ name: Build
 on:
   push:
     branches:
-      - main
+      - master
```

This should be `main`
```suggestion
      - main
```